src/components/AuthInput/AuthInput.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import "./AuthInput.scss";
+import { InputProps } from "../../types/componentsPropsTypes/InputProps";
+
+const AuthInput = ({
+  labelName,
+  htmlFor,
```

Comment:
I don't think you need this, cause it will be the same as ```id```
and I think u can do it without id and htmlFor, label will link to nearest input automatically